docs/ql-documentation/support/read-me-project.txt

```diff
@@ -0,0 +1,18 @@
+Publishing this project for a new version
+#########################################
+
+To update this project for a new version:
+
+#. Check with the language teams that all information in the ``ql/change-notes/support/`` directory is ready.
+
+#. Check that the versions of these files in the ``code`` repository match those in the ``ql`` repository.
+That is, there has been a "ql submodule bump" in the ``code`` repository since the last change to those files.
```

Comment:
This step can be deleted and we should probably update the file format. Feel free to raise a Jira issue for me since I wrote it.